.zuul.yaml

```diff
@@ -3,16 +3,19 @@
     name: functional-autoscaling-tests-osp18
     parent: telemetry-operator-multinode-autoscaling
     description: |
-      Run the autoscaling functional tests and tempest tests on osp18+patched
-      versions of aodh and heat.
+      Run autoscaling funtional and tempest tests and the metrics functional tests.
```

Comment:
If this job and the functional-metrics-verification job are now the same, you can redefine the functional-metric-verification-tests-osp18 job to be an alias for this one

This skips the need to keep both around until the telemetry-operator removes `functional-metric-verification-tests-osp18` and then we can remove all the aliases in one go later.

It's possible that we need to keep these defined since telemetry-operator on the FR2 branch still uses the old job names. :/ I am not sure how we will approach that long-term. I think we'll have to backport the zuul config changes to telemetry-operator/FR2.